platform-core-module/src/test/java/com/softicar/platform/core/module/program/execution/scheduled/ScheduledProgramEnqueuerTest.java

```diff
@@ -4,12 +4,12 @@
 import com.softicar.platform.common.date.DayTime;
 import com.softicar.platform.common.date.Time;
 import com.softicar.platform.core.module.program.AGProgram;
+import com.softicar.platform.core.module.program.AbstractProgramTest;
 import com.softicar.platform.core.module.program.execution.AGProgramExecution;
-import com.softicar.platform.db.runtime.test.AbstractDbTest;
 import java.util.UUID;
 import org.junit.Test;
 
-public class ScheduledProgramEnqueuerTest extends AbstractDbTest {
+public class ScheduledProgramEnqueuerTest extends AbstractProgramTest {
```

Comment:
I think we need to modify `assertQueuedAt` below, as follows:
- rename it to `assertQueued`
- also let it check the `QUEUED_BY` field

Like so:
```
private void assertQueued(DayTime expectedQueuedAt) {

	assertEquals(expectedQueuedAt, program.getQueuedAt());
	assertEquals(expectedQueuedAt != null? /*system user*/ : null, program.getQueuedBy());
}
```